### ISSUE TITLE
Darkmoon Faire spawn???

### ISSUE THREAD

**Issue by lacerbeamz** [dalaranwow/dalaran-wow#4378]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I have been waiting for the darkmoon faire since the start of the month... i check all 3 spawns daily and still have yet to see a single person aside from one Nelf carnie in mulgore but no faire
**Expected behaviour**: Tell us what should happen instead
to have an actual darkmoon faire so i can turn in my best in slot trinket 
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Domynator** [comment]
I have been checking for the last 3 days in all 3 locations, no darkmoon faire

**Comment by Rushor** [comment]
current timer from ptr which work fine:

```sql
INSERT INTO `game_event` (`eventEntry`, `start_time`, `end_time`, `occurence`, `length`, `holiday`, `description`, `world_event`, `announce`) VALUES (3, '2013-02-03 01:01:00', '2020-12-31 06:00:00', 131040, 8639, 376, 'Darkmoon Faire (Terokkar Forest)', 0, 2);
INSERT INTO `game_event` (`eventEntry`, `start_time`, `end_time`, `occurence`, `length`, `holiday`, `description`, `world_event`, `announce`) VALUES (4, '2013-03-03 01:01:00', '2020-12-31 06:00:00', 131040, 8639, 374, 'Darkmoon Faire (Elwynn Forest)', 0, 2);
INSERT INTO `game_event` (`eventEntry`, `start_time`, `end_time`, `occurence`, `length`, `holiday`, `description`, `world_event`, `announce`) VALUES (5, '2013-01-06 01:01:00', '2020-12-31 06:00:00', 131040, 8639, 375, 'Darkmoon Faire (Mulgore)', 0, 2);
INSERT INTO `game_event` (`eventEntry`, `start_time`, `end_time`, `occurence`, `length`, `holiday`, `description`, `world_event`, `announce`) VALUES (23, '2011-03-03 01:01:00', '2020-12-31 06:00:00', 131040, 4320, 0, 'Darkmoon Faire Building (Elwynn Forest)', 0, 2);
```

moreover, duplicate of https://github.com/dalaranwow/dalaran-wow/issues/2849
